main.py

```diff
@@ -30,7 +31,36 @@ def Remove(self):
     def Add(self):
         if self._Rear < self._MaxSize - 1:
             TileLibrary = CreateTileLibrary()
-            RandNo = random.randint(0, 26)
+            tile_frequency = [
```

Comment:
Perhaps list these all on the same line? Or at least move the final brace up a line.